ui/src/layout/RuntimeScan/ScanConfiguration/ScanOptionsForm.js

```diff
@@ -36,7 +36,7 @@ const ScanOptionsForm = ({onClose}) => {
                     })}
                 >
                     <ToggleField name="cisDockerBenchmarkScanEnabled" label="CIS Docker Benchmark" />
-                    <TextField name="maxScanParallelism" label="Max Scan Parallelism" type="number" />
+                    <TextField name="maxScanParallelism" label="Max Scan Parallelism" type="number" min="0" />
```

Comment:
1?

Reply:
In the backend what MaxScanParallelism means is "maximum number of concurrent scanners running" so 0 means that nothing would get scanned. Perhaps this is bad wording on our part and this field needs to be renamed "Max Concurrent Scanners" or similar? WDYT @akpsgit ?

Reply:
The API defined it as min 1, so we should use also 1 in the js code I think
https://github.com/openclarity/kubeclarity/blob/main/api/swagger.yaml#L719